cbcontainers/models/compatibility_matrix.go

```diff
@@ -0,0 +1,55 @@
+package models
+
+import "fmt"
+
+type AgentVersion string
+
+const (
+	AgentMinVersionNone   AgentVersion = "none"
+	AgentMaxVersionLatest AgentVersion = "latest"
+	AgentVersionUnknown   AgentVersion = ""
+)
+
+func (v AgentVersion) IsBiggerThan(version string) bool {
```

Comment:
Shouldn't `latest` be bigger than all ? 

Also I think `Larger` is more appropriate term (less <-> large, small <-> big)

Reply:
in this context we don't care about `latest`. we only care if `v` (which is the minimum) is bigger/larger than `version` which means we trying to install too old agent version